dataflux_pytorch/benchmark/README.md

```diff
@@ -94,9 +94,9 @@ Dataflux's implementation of CheckpointIO for PyTorch Lightning is undergoing ac
    </td>
```

Comment:
Could you also add a description at the top of the table about the VM size you were using for this, and the fact that it and the bucket are in the same region? So that anyone who sees these benchmarks and tries to reproduce them doesn't hit any issues with those things